pom.xml

```diff
@@ -120,6 +120,10 @@
       <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>elementary-releases</id>
+      <url>https://repo.karuslabs.com/repository/elementary-releases/</url>
+    </repository>
```

Comment:
This is only pending https://github.com/jenkins-infra/helpdesk/issues/3115 right?